### PR TITLE
node: consolidate python installation to two layers

### PR DESCRIPTION
**Pull request by grayside**
Removes unnecessary steps and consolidates to fewer layers.

### CODE REVIEW

File: node/12-user/Dockerfile [hunk 1]
```diff
@@ -16,14 +16,12 @@ FROM node:12-stretch
...
-RUN echo 'export PATH="/home/node/.pyenv/bin:$PATH"' >> ~/.profile
-RUN echo 'export PATH="/home/node/.pyenv/shims:$PATH"' >> ~/.profile
-RUN echo 'eval "$(pyenv init -)"' >> ~/.profile
-RUN echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
-
+# Install Python
 ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
-
-RUN pyenv install 3.7.4
-RUN pyenv global 3.7.4
```
Comment (grayside) [inline node/12-user/Dockerfile]: Ah, I see node 12 is coupled with Python 3.7.4, should they all be running Python 3.7.4 or should this one be on a forward patch?

Comment (bcoe) [reply node/12-user/Dockerfile]: We should check with @busunkim96, we use `python3` in a couple places in our Node.js applications, here:

https://github.com/googleapis/nodejs-firestore/blob/master/.kokoro/release/docs.sh#L32

and, here:

https://github.com/googleapis/nodejs-firestore/blob/master/.kokoro/publish.sh#L22

I believe there were some restrictions on which Python version could run the tooling in question.

**Comment by JustinBeckwith** [comment]
@bcoe @grayside coming back around - is this a change we want to land?  I am scared we have no tests in this repository.

**Comment by bcoe** [comment]
@busunkim96 do you know if there are any constraints on the python version used by `releasetool` or the doc upload service?

**Comment by busunkim96** [comment]
@bcoe >=3.6 is the only requirement.  https://github.com/googleapis/releasetool/blob/master/setup.py#L58 and

**Comment by parthea** [comment]
@bcoe @grayside  I'm going to close this PR due to inactivity but please feel free to re-open it if it's still needed.

**Pull request closed without merging**